test/debug/break_test.rb

```diff
@@ -8,29 +8,29 @@ module DEBUGGER__
   #
   class BreakAtMethodsTest < TestCase
     def program
-      <<~RUBY
-        module Foo
-          class Bar
-            def self.a
-              "hello"
-            end
-
-            def b(n)
-              2.times do
-                n
-              end
-            end
-          end
-          module Baz
-            def self.c
-              1
-            end
-          end
-          Bar.a
-          bar = Bar.new
-          bar.b(1)
-          Baz.c
-        end
+      strip_line_num <<~RUBY
```

Comment:
for consistency reason, we may want **all** program code to have line numbers. and if that makes sense to you, maybe we can strip the line at a single place (perhaps in `debug_code`)? then we won't need to call this in every `program` method.

Reply:
I agree with your opinion. Thank you!
https://github.com/ruby/debug/pull/42/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R22